src/main/webapp/index.html

```diff
@@ -157,19 +156,19 @@ <h4>Career Pathways</h4>
           <a href="#link2">What is a Web Developer</a>
           <ul class="resource-link">
             <li>
-              <a href="https://bit.ly/2AaTXon" target="_blank">Defenition and Details</a>
+              <a href="https://bit.ly/2AaTXon" target="_blank">Definition and Details</a>
             </li>
             <li>
-              <a href="https://bit.ly/2VrjH7v" target="_blank">Udemy: Web Developer Trainings</a>
+              <a href="https://bit.ly/2VrjH7v" target="_blank">Udemy: Web Developer trainings</a>
             </li>
             <li>
-              <a href="https://bit.ly/2Vs2bju" target="_blank">Udemy: Personal Pick for Web Development</a>
+              <a href="https://bit.ly/2Vs2bju" target="_blank">Udemy: Personal pick for Web Development</a>
```

Comment:
Personal picks for Web Development

Reply:
It only links to one course that I took and personally enjoyed